internal/usage/usage.go

```diff
@@ -249,9 +249,9 @@ dbName and collection are required only for built-in roles.`
 	LinkTokenAccessListCIDREntries            = "IP address access list entries that are associated with the link-token." //nolint:gosec // This is just a message not a credential
 	LinkToken                                 = "Link-token generated by Atlas."                                          //nolint:gosec // This is just a message not a credential
 	LiveMigrationID                           = "Unique 24-hexadecimal digit string that identifies the live migration job."
-	PrivateEndpointID                         = "Unique identifier of the AWS PrivateLink connection."
-	EndpointServiceID                         = "Unique identifier of the private endpoint service for which you want to retrieve a private endpoint."
-	PrivateEndpointIDAzure                    = "Unique identifier of the Azure private endpoint resource."
+	PrivateEndpointID                         = "Unique 22-character alphanumeric string that identifies the AWS PrivateLink connection in AWS. You can find this value on the AWS VPC Dashboard under Endpoints > VPC ID."
```

Comment:
Question: Should this be a 24-character? 
```suggestion
	PrivateEndpointID                         = "Unique 24-character alphanumeric string that identifies the AWS PrivateLink connection in AWS. You can find this value on the AWS VPC Dashboard under Endpoints > VPC ID."
```

Reply:
This is actually the AWS ID, which is 22 characters (looks like this: vpce-00713b5e644e830a3), so it should be 22